modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -440,11 +457,47 @@ public void testSendNotificationWithAccountRoles() throws Exception {
 
 		// Send email with an object definition not restricted by account entry
 
+		User user4 = UserTestUtil.addUser();
+
+		Organization organization1 = _organizationLocalService.addOrganization(
+			TestPropsValues.getUserId(),
+			OrganizationConstants.DEFAULT_PARENT_ORGANIZATION_ID,
+			RandomTestUtil.randomString(), false);
+
+		_userGroupRoleLocalService.addUserGroupRole(
+			user4.getUserId(), organization1.getGroupId(),
+			organizationRole1.getRoleId());
+
+		_userGroupRoleLocalService.addUserGroupRole(
+			user3.getUserId(), organization1.getGroupId(),
+			organizationRole1.getRoleId());
+
+		User user5 = UserTestUtil.addUser();
+
+		Organization parentOrganization =
+			_organizationLocalService.addOrganization(
+				TestPropsValues.getUserId(),
+				OrganizationConstants.DEFAULT_PARENT_ORGANIZATION_ID,
+				RandomTestUtil.randomString(), false);
+
+		Organization organization2 = _organizationLocalService.addOrganization(
+			TestPropsValues.getUserId(), parentOrganization.getOrganizationId(),
+			RandomTestUtil.randomString(), false);
```

Comment:
@devpedrotavares let's simplify the organization variable names to something like:

```
Organization parentOrganization1 = ...

Organization parentOrganization2 =

Organization childOrganization = 
```

Reply:
There is only one parent organization, so I did:

```
Organization organization1

Organization organization2

Organization childOrganization
```
Is that okay?